requirements.txt

```
@@ -29,7 +29,7 @@ escapism==1.*
 # Used for GCP billing management
 pandas
 # FIXME: Unpin when https://github.com/unionai-oss/pandera/issues/1656 gets solved
-numpy<2
+numpy<3
```

Just because this reads a bit easier.

```suggestion
numpy==2.*
```